lib/miq/lwimiq.rb

```diff
@@ -23,14 +27,15 @@ def generate
     end
 
     def path
-      File.join("site", "_posts", "#{Date.today.strftime("%Y-%m-%d")}-CHANGE-ME.md")
+      File.join("site", "_posts", "#{Date.today.strftime("%Y-%m-%d")}-#{file_rename}.md")
```

Comment:
`file_rename` was a little confusing to me. WDYT about pulling the whole of this last part out as `#basename`?

Reply:
@imtayadeway - e.g. calling the variable `basename`?